install_config/master_node_configuration.adoc

```diff
@@ -1284,6 +1290,8 @@ that group.
 <5> A list of groups the rule applies to. An empty list implies every group.
 <6> A list of non-resources URLs the rule applies to.
 <7> A list of namespaces the rule applies to. An empty list implies every namespace.
+<8> Endpoint used by the web console.
```

Comment:
Is this enough information for a user to construct the endpoints?

Reply:
Not sure, what else we can add here. It's basically an information that this endpoint is where the logging from web console are going through.